salt/proxy/nxos.py

```diff
@@ -402,7 +401,7 @@ def _init_ssh(opts):
         log.error('-- Verify that "feature ssh" is enabled on your NX-OS device: %s', opts['proxy']['host'])
         log.error('-- Exception Generated: %s', ex)
         log.error(ex)
-        exit()
+        raise
```

Comment:
@dhiltonp Curious about this change.  Without the `exit()` we will see the message but the proxy minion will remain in an initialization / connection loop trying over and over to establish a connection making the message really hard to read.

Reply:
With exit(), a ssh connection issue prevents the salt minion from starting up at all (I think). Am I wrong?

Reply:
No, you are not wrong but that was the intent.  We did not want the minion to start up if there was an ssh connection issue but exit with guidance about how to fix the problem and then restart the minion.

Reply:
`exit()` doesn't return a special error code or anything. We'd need to at least change this to be `exit(1)`, but raise is much preferred by the team.

Reply:
@mikewiebe I've made just about all the changes I know should be made, could you do some manual/integration tests against nxos.